threshsign/src/VectorOfShares.cpp

```diff
@@ -183,7 +181,7 @@ void VectorOfShares::fromBytes(const unsigned char * buf, int len) {
         for(int c = 0; c < 8; c++) {
             if(byte & bitMask) {
                 int id1 = b * 8 + (c+1);
-                //LOG_DEBUG(GL, "bitMask = " << bitMask << ", deserialized ID " << id1 << " from byte " << (int)byte);
+                //LOG_TRACE(GL, "bitMask = " << bitMask << ", deserialized ID " << id1 << " from byte " << (int)byte);
```

Comment:
It seems odd to comment out trace messages. If we intend to ever use them, we'd likely have tracing on.

Reply:
Still, there's at least one unnecessary 'if' there

Reply:
I don't follow.

Reply:
every log message, even if disabled, performs if(debug_level <= current_level)

Reply:
Ok, so it's a performance related change. Can you put a comment above the line stating that please.

Reply:
it was commented out, but in LOG_DEBUG, not LOG_TRACE

Reply:
Ok. I guess we can leave it for now. Logging in a tight loop probably doesn't make sense anyway and can likely be replaced with a pretty simple unit test in the future.